test/routes/technicalQualification.route.test.js

```diff
@@ -78,8 +78,22 @@ lab.experiment('Technical Management Qualification tests:', () => {
       Code.expect(doc.getElementById('submit-button').firstChild.nodeValue).to.equal('Continue')
       Code.expect(doc.getElementById('form').getAttribute('action')).to.equal(routePath)
 
+      // Test static html exists
```

Comment:
Rather than having all this repeated 'doc.getElementById...' code in my tests I have simply defined an array of expected IDs and then looped through the array

e.g.

  const elementIds = [
    'back-link',
    'defra-csrf-token',
    'postcode-label',
    'postcode-hint',
    'manual-hint',
    'manual-address-link',
    'no-postcode-link-text'
  ]
  for (let id of elementIds) {
    element = doc.getElementById(id)
    Code.expect(doc.getElementById(id)).to.exist()
  }

That way, if you want to add / remove an element all you have to do is add or remove the ID in the array.